src/component/TodoItem.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react';
+import Todos from '../mock/data';
+
+class TodoItem extends Component {
+  render() {
+    const TodoList = Todos.map(todo => (
```

Comment:
### Critical

Nopppppeeeee!!!!
Define this as a component on its own. This is not ideal.